ast/definition.go

```diff
@@ -298,6 +298,7 @@ const (
 //
 type Field struct {
 	ID           int
+	IDUnset      bool
```

Comment:
Since this one is unique, it might be worth adding a quick comment here.

```suggestion
	// IDUnset specifies that a field identifier was not
	// provided for this field.
	IDUnset      bool
```